recipes/pyextremes/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyextremes" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8732597818a5f2ee7bcb2b3c742aae5ec43a25957f214713278884a95b2eb8ef
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - numpy >=1.18.1
+    - scipy >=1.4.1
+    - pandas >=1.0.1
+    - matplotlib-base >=3.1.0
+    - emcee >=2.2.1
```

Comment:
Most of these are probably not host requirements needed to build the package. Should be just python and pip